includes/class-wc-session-handler.php

```diff
@@ -187,6 +187,25 @@ public function generate_customer_id() {
 		return $customer_id;
 	}
 
+	/**
+	 * Get session unique ID for quests if session is initialized or user ID if logged in.
+	 * Introduced to help with unit tests.
+	 *
+	 * @since 5.3.0
+	 * @return string
+	 */
+	public function get_customer_unique_id() {
+		$customer_id = '';
+
+		if ( $this->has_session() && $this->_customer_id ) {
+			$customer_id = $this->_customer_id;
+		} elseif ( is_user_logged_in() ) {
+			$customer_id = (string) get_current_user_id();
+		}
+
+		return $customer_id;
```

Comment:
Return type is stated as string and only line 203 enforces that even though 201 may already be string. So perhaps remove `(string)` from line 203 and put it on line 206 `return (string) $customer_id` would be better?

Reply:
Only `get_current_user_id()` returns `int`, that's why I didn't want to cast string for everything by default.

Reply:
See `generate_customer_id()`, there even if we use `get_current_user_id()` we still cast string.